### DISCUSSION TITLE
Microsoft Azure SSO Help

### DISCUSSION THREAD

**Discussion by bDestroyerd** [9p4/jellyfin-plugin-sso#208] [Q&A]
Hello, I need help with setting up SSO to work with Microsoft Azure/Entra. I couldn't find anything on Google, Reddit, YouTube, or here on the GitHub page. I should know everything that I need to to set it up on Azure, so all I should need help with is the setting up on Jellyfin with SAML so that it links with Azure.
I use Cloudflare and NGINX Proxy Manager to traffic my data. And if it is of any importance, I run Jellyfin and NGINX on my UnRAID server. If you need anything else from me than, or if this already has an answer somewhere that I couldn't find, then I will be more than happy to oblige. Thank you regardless.

**Comment by 9p4** [comment]
Unfortunately, I do not have access to Azure SSO to test this out with.
